ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRao.java

```diff
@@ -45,9 +48,35 @@ public String getVersion() {
             throw new FaraoException("There are some issues in RAO parameters:" + System.lineSeparator() + String.join(System.lineSeparator(), configQualityCheck));
         }
 
+        // compute maximum loop flow value F_(0,all)_MAX, and update it for each Cnec in Crac
+        FlowBasedComputationParameters flowBasedComputationParameters = parameters.getExtension(FlowBasedComputationParameters.class);
+        calculateLoopFlowConstraintAndUpdateAllCnec(network, crac, computationManager, flowBasedComputationParameters);
+
```

Comment:
We cannot merge that, otherwise it will broke our existing integration tests, with Crac without loopflow extensions.

In the long-term : the config will inform on whether we compute loopflow or not.
In the meantime : we could do something like `if the crac has a loopflow extension then calculate loopflow else do nothing`

Reply:
ok